src/app/modules/superNodeProgram/superNodeProgram.controller.js

```diff
@@ -295,7 +295,7 @@ class SuperNodeProgramCtrl {
             })
         }
 
-        if(!isEnrolled && isEnrollmentAddressValid && 64 === codewordHash.length) {
+        if((!isEnrolled || isEnrolled && this.formData.domainOrIP !== this.enrollStatus.endpoint) && isEnrollmentAddressValid && 64 === codewordHash.length) {
```

Comment:
Since you are using twice comparasion to domainOrIp I think it could be change like this:

```
 const isEnrolledWithSameDomain = isEnrolled && this.formData.domainOrIP === this.enrollStatus.endpoint

if (isEnrolledWithSameDomain) {
            this._$timeout(() => {
                this._Alert.addressEnrolled();
                this.okPressed = false;
                return;
            })
}

....

if(!isEnrolledWithSameDomain && isEnrollmentAddressValid && 64 === codewordHash.length) {

...

```